### PR TITLE
Add wazuh version file to wazuh dashboard and wazuh indexer

### PR DESCRIPTION
**Pull request by rauldpm**
|Related issue|
|---|
|Closes https://github.com/wazuh/wazuh-packages/issues/1614 |

## Description

- This PR adds the VERSION file to the wazuh dashboard and wazuh indexer package, this PR fixes some variables and nomenclature naming and format.

## Logs example

- See issue comments

## Tests

- Build the package in any supported platform
  - [x] Linux
- [x] Package installation
- [x] Package upgrade
- [x] Package remove
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [x] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [x] Package install/purge/install
  - [x] Check file permissions after installing the package

### CODE REVIEW

**Review by c-bordon — APPROVED** [review]
LGTM

**Comment by rauldpm** [comment]
### Update report

- Generated packages with future using the packages builder pipeline

<details><summary>Upgrade wazuh dashboard rpm</summary>

```
[root@wazuh-server wazuh-user]# ls -l /usr/share/wazuh-dashboard/
total 1172
drwxr-x---.   2 wazuh-dashboard wazuh-dashboard     109 Jun 14 12:37 bin
drwxr-x---.   2 wazuh-dashboard wazuh-dashboard      44 Jun 14 12:38 config
drwxr-x---.   3 wazuh-dashboard wazuh-dashboard      31 Jun 14 12:38 data
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard   11358 Nov 22  2021 LICENSE.txt
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard    3098 Nov 22  2021 manifest.yml
drwxr-x---.   6 wazuh-dashboard wazuh-dashboard     108 Jun 14 12:37 node
drwxr-x---. 703 wazuh-dashboard wazuh-dashboard   20480 Jun 14 12:37 node_modules
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard 1137439 Nov 22  2021 NOTICE.txt
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard     827 Nov 22  2021 package.json
drwxr-x---.   8 wazuh-dashboard wazuh-dashboard     157 Jun 14 12:38 plugins
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard    1925 Nov 22  2021 README.txt
drwxr-x---.  11 wazuh-dashboard wazuh-dashboard     160 Jun 14 12:38 src
[root@wazuh-server wazuh-user]# yum localinstall wazuh-dashboard-99.99.0-1.x86_64.rpm -y
Loaded plugins: fastestmirror
Examining wazuh-dashboard-99.99.0-1.x86_64.rpm: wazuh-dashboard-99.99.0-1.x86_64
Marking wazuh-dashboard-99.99.0-1.x86_64.rpm as an update to wazuh-dashboard-4.3.4-1.x86_64
Resolving Dependencies
--> Running transaction check
---> Package wazuh-dashboard.x86_64 0:4.3.4-1 will be updated
---> Package wazuh-dashboard.x86_64 0:99.99.0-1 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

===================================================================================================================================================================================================================
 Package                                           Arch                                     Version                                      Repository                                                           Size
===================================================================================================================================================================================================================
Updating:
 wazuh-dashboard                                   x86_64                                   99.99.0-1                                    /wazuh-dashboard-99.99.0-1.x86_64                                   588 M

Transaction Summary
===================================================================================================================================================================================================================
Upgrade  1 Package

Total size: 588 M
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : wazuh-dashboard-99.99.0-1.x86_64                                                                                                                                                                1/2 
  Cleanup    : wazuh-dashboard-4.3.4-1.x86_64                                                                                                                                                                  2/2 
  Verifying  : wazuh-dashboard-99.99.0-1.x86_64                                                                                                                                                                1/2 
  Verifying  : wazuh-dashboard-4.3.4-1.x86_64                                                                                                                                                                  2/2 

Updated:
  wazuh-dashboard.x86_64 0:99.99.0-1                                                                                                                                                                               

Complete!
[root@wazuh-server wazuh-user]# ls -l /usr/share/wazuh-dashboard/
total 1176
drwxr-x---.   2 wazuh-dashboard wazuh-dashboard     109 Jun 14 16:51 bin
drwxr-x---.   2 wazuh-dashboard wazuh-dashboard      44 Jun 13 21:11 config
drwxr-x---.   3 wazuh-dashboard wazuh-dashboard      31 Nov 22  2021 data
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard   11358 Nov 22  2021 LICENSE.txt
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard    3098 Nov 22  2021 manifest.yml
drwxr-x---.   6 wazuh-dashboard wazuh-dashboard     108 Jun 14 16:51 node
drwxr-x---. 703 wazuh-dashboard wazuh-dashboard   20480 Jun 14 16:51 node_modules
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard 1137439 Nov 22  2021 NOTICE.txt
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard     827 Nov 22  2021 package.json
drwxr-x---.   8 wazuh-dashboard wazuh-dashboard     157 Jun 14 16:44 plugins
-rw-r-----.   1 wazuh-dashboard wazuh-dashboard    1925 Nov 22  2021 README.txt
drwxr-x---.  11 wazuh-dashboard wazuh-dashboard     160 Jun 14 16:51 src
-r--r-----.   1 wazuh-dashboard wazuh-dashboard       6 Jun 13 21:11 VERSION
[root@wazuh-server wazuh-user]# ls -l /usr/share/wazuh-dashboard/VERSION 
-r--r-----. 1 wazuh-dashboard wazuh-dashboard 6 Jun 13 21:11 /usr/share/wazuh-dashboard/VERSION
[root@wazuh-server wazuh-user]#
[root@wazuh-server wazuh-user]# cat /usr/share/wazuh-dashboard/VERSION 
4.3.5
[root@wazuh-server wazuh-user]#
```

</details>

File: stack/dashboard/deb/build_package.sh [hunk 1]
```diff
@@ -35,51 +35,51 @@ ctrl_c() {
...
-            ${FUTURE} ${BASE} ${REFERENCE} || return 1
+    volumes="-v ${outdir}/:/tmp:Z"
+    if [ "${reference}" ];then
+        docker run -t --rm ${volumes} \
+            ${container_name} ${architecture} ${revision} \
+            ${future} ${base} ${reference} || return 1
     else
-        if [ "${BASE}" = "local" ];then
-            VOLUMES="${VOLUMES} -v ${BASE_PATH}:/root/output:Z"
+        if [ "${base}" == "local" ];then
```
Comment (okynos) [inline stack/dashboard/deb/build_package.sh:57]: Same

File: stack/indexer/base/builder.sh [hunk 2]
```diff
@@ -10,28 +10,43 @@
...
 # Including files
 if [ "${reference}" ];then
     curl -sL https://github.com/wazuh/wazuh-packages/tarball/"${reference}" | tar xz
     cp -r ./wazuh*/* /root/
+    version=$(curl -sL https://raw.githubusercontent.com/wazuh/wazuh-packages/${reference}/VERSION | cat)
+else
+    version=$(cat /root/VERSION)
+fi
+
+if [ "${future}" == "yes" ];then
```
Comment (okynos) [inline stack/indexer/base/builder.sh:34]: Same

**Review by okynos — CHANGES_REQUESTED** [review]

File: stack/indexer/base/builder.sh (hunk 2 above)
Comment (rauldpm) [reply stack/indexer/base/builder.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/1648/commits/88b143bb8737d8b65fb27be9fe101805a0fe3d7a

File: stack/dashboard/deb/build_package.sh (hunk 1 above)
Comment (rauldpm) [reply stack/dashboard/deb/build_package.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/1648/commits/88b143bb8737d8b65fb27be9fe101805a0fe3d7a

**Review by okynos — APPROVED** [review]

**Pull request merged**